ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/recaptcha/v1/recaptcha/clientlibs/editor/js/editDialog.js

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ * Copyright 2022 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+(function($) {
+    "use strict";
+
+    var EDIT_DIALOG = ".cmp-adaptiveform-recaptcha__editdialog",
+        RECAPTCHA_CONFIG = EDIT_DIALOG + " .cmp-adaptiveform-recaptcha__configuration",
+        RECAPTCHA_SIZE = EDIT_DIALOG + " .cmp-adaptiveform-recaptcha__size",
+        RECAPTCHA_SIZE_NORMAL_RADIO_BTN = "coral-radio[name='./recaptchaSize'][value='normal']",
```

Comment:
Can we make use of BEM classes and avoid dependency on coral selectors

Reply:
Modified as requested